src/signals/incident-management/components/Label/__test__/Label.test.js

```diff
@@ -5,14 +5,14 @@ import 'jest-styled-components';
 
 import Label from '../';
 
-describe.skip('signals/incident-management/components/Label', () => {
+describe('<Label /', () => {
   afterEach(cleanup);
 
   it('should render a label element', () => {
     const { container, getByLabelText } = render(
       withAppContext(
         <div>
-          <Label htmlFor="someOtherElementId">This is my label text</Label>
+          <Label isGroupHeader={false} htmlFor="someOtherElementId">This is my label text</Label>
```

Comment:
You can omit each occurrence of `isGroupHeader={false}` since the prop's default value is 'false'.

Reply:
yes that is true